src/test/java/org/opensearch/flowframework/exception/ApiSpecParseExceptionTests.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.flowframework.exception;
+
+import org.opensearch.test.OpenSearchTestCase;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class ApiSpecParseExceptionTests extends OpenSearchTestCase {
+
+    public void testApiSpecParseException() {
+        ApiSpecParseException exception = new ApiSpecParseException("API spec parsing failed");
+        assertTrue(true);
+        assertEquals("API spec parsing failed", exception.getMessage());
+    }
+
+    public void testApiSpecParseExceptionWithCause() {
+        Throwable cause = new RuntimeException("Underlying issue");
+        ApiSpecParseException exception = new ApiSpecParseException("API spec parsing failed", cause);
+        assertTrue(true);
```

Comment:
![image](https://github.com/user-attachments/assets/c7a8ac52-4dc4-4ca2-aa7c-2317dcf7cbb9)

Reply:
Haha, I just realized how funny this is. This will be the last time I use a code refactor suggestion without thinking it through.
<img width="587" alt="Screenshot 2024-10-07 at 1 48 19 PM" src="https://github.com/user-attachments/assets/f01cd1b4-4620-4553-ac47-c3eaf93a7c9b">